optimum/intel/neural_compressor/quantization.py

```diff
@@ -374,22 +374,33 @@ def _weight_only_quantization(
     }
 
     low_cpu_mem_usage = True
+
     if use_xpu:
-        try:
-            # TODO: if low_cpu_mem_uasge is True, gptj will have accuracy issue on CPU device.
-            model = model_class.from_pretrained(
-                model_id, low_cpu_mem_usage=low_cpu_mem_usage, device_map="cpu", **loading_kwargs
-            )
-        except NotImplementedError:
-            logger.info(
-                "Failed to load models with `low_cpu_mem_usage=True`, will fall to traditional load method resulting in higher memory consumption."
-            )
-            low_cpu_mem_usage = False
-            model = model_class.from_pretrained(model_id, low_cpu_mem_usage=low_cpu_mem_usage, **loading_kwargs)
+        if hasattr(quantization_config, "use_layer_wise") and quantization_config.use_layer_wise:
+            from neural_compressor.torch import load_empty_model
+
+            model = load_empty_model(model_id, cls=model_class, trust_remote_code=trust_remote_code)
```

Comment:
why not :
```suggestion
            model = load_empty_model(model_id, cls=model_class, **loading_kwargs)
```

Reply:
Let me make explaination, because we initialize the config first, and then initialize the model in `load_empty_model` function. https://github.com/intel/neural-compressor/blob/e2696603f45f5796f1c048aab33eef11aaeb2cdb/neural_compressor/adaptor/torch_utils/layer_wise_quant/utils.py#L122
when pass the `**loading_kwargs`, config initialize will raise error like the following.
```
>           model = cls(config, **kwargs)
E           TypeError: __init__() got an unexpected keyword argument 'subfolder'
```

Reply:
improved it in INC code, there is no issue now.

Reply:
I don't understand why is the empty model created using `from_config` and not `from_pretrained` ? 
the context manager you're using `with init_empty_weights()` works with `from_pretrained` which accepts all kwargs.

Reply:
I yesterday raised PR to improve it in INC and the PR has merged, I also update commit-id in optimum-inte setup.py, `load_empty_model` can support pass `**loading_kwargs` and works with `from_pretrained` now. the issue no longer exists. :)
Here is the INC PR https://github.com/intel/neural-compressor/pull/2079.